brax/training/networks.py

```diff
@@ -82,48 +83,55 @@ def __call__(self, data: jnp.ndarray):
         hidden = self.activation(hidden)
     return hidden
 
+def canolicalize_obs_size(obs_size: types.ObservationSize) -> int:
+    obs_size = obs_size['state'] if isinstance(obs_size, Mapping) else obs_size
+    return tree_flatten(obs_size)[0][-1] # Handle tuple and int size specs.
 
 def make_policy_network(
     param_size: int,
-    obs_size: int,
+    obs_size: types.ObservationSize,
     preprocess_observations_fn: types.PreprocessObservationFn = types
     .identity_observation_preprocessor,
     hidden_layer_sizes: Sequence[int] = (256, 256),
     activation: ActivationFn = linen.relu,
     kernel_init: Initializer = jax.nn.initializers.lecun_uniform(),
     layer_norm: bool = False) -> FeedForwardNetwork:
-  """Creates a policy network."""
+  """Creates a policy network. Only processes state in the case of dict obs."""
```

Comment:
nit: this comment is better suited where it's applied rather than in the top-level function docstring. So on L107

Similar comment for value_network below